OpenDataPolicingNC/Traffic-Stops
# Support ACS 2016 5-Year - WIP

**WIP - DO NOT MERGE**

* Update Python census package to latest version, which supports ACS 2016, and our API code to match
* Update race variable names to be the non-hispanic versions to match the current presentation
* Update existing GEO_ID pairings since 2016 added 00s to the values (16000US3719000 became 1600000US3719000)

To test:
* Run ```python manage.py import_census```
* This will update everything in http://localhost:8000/admin/tsdata/censusprofile/ to be ACS 2016

To test a specific city:

* Slow way: run importer again for NC/MD
* Quick way: Go to http://localhost:8000/admin/nc/agency/?o=1 and find an agency with a census profile id. Manually add 00 to the ID: 16000US3702140 becomes 1600000US3702140

Then:

* Find that ID on http://localhost:8000/admin/tsdata/censusprofile/ and make note of the **Place Name** (e.g. Asheville city, North Carolina)
* Make sure the values display correctly on http://localhost:8000/nc/api/agency/
* Visit https://factfinder.census.gov/faces/nav/jsf/pages/index.xhtml and search for the **Place Name** above
   * On the next page, click **Race and Hispanic Origin** in the left nav
   * On the next page, click **Demographic and Housing Estimates (Age, Sex, Race, Households and Housing, ...)** under 2016 American Community Survey
   * Our values should match the values under **HISPANIC OR LATINO AND RACE**
